### DISCUSSION TITLE
Why is this happening in my code。为什么我的代码会是这样子

### DISCUSSION THREAD

**Discussion by angluds** [community/community#119947] [Programming Help]
### Hi everyone,

![屏幕截图 2024-04-17 205937](https://github.com/community/community/assets/148355312/90dc3841-1baf-408e-9487-30dff1ccbc5d)
![屏幕截图 2024-04-17 210100](https://github.com/community/community/assets/148355312/c3d413fc-5143-4dfa-8bb6-fc65780489d4)
LIN=3,         COL=3

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @angluds , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by iqnite** [comment]
The function you provided looks OK to me, the question marks are there because the Terminal can't display emojis and some special characters.
